MetricsUploader/MetricsUploaderLinux.cpp

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "MetricsUploader/MetricsUploader.h"
+
+namespace orbit_metrics_uploader {
+
+MetricsUploader::~MetricsUploader() = default;
+
+ErrorMessageOr<std::unique_ptr<MetricsUploader>> MetricsUploader::CreateMetricsUploader([
+    [maybe_unused]] std::string_view client_name) {
```

Comment:
The line break looks weird. Seems to be a clang-format problem. I'd suggest to not specify the argument name (client_name). Then the attribute is not needed in the first place.

Reply:
Yes, strange for me as well. Removed argument name and it looks better now